pom.xml

```diff
@@ -62,11 +62,10 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <revision>3.0-java11-alpha-2</revision>
+        <revision>3.0</revision>
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.121.1</jenkins.version>
         <java.level>8</java.level>
-        <plugin.minimumJavaVersion>11</plugin.minimumJavaVersion>
```

Comment:
The code will work fine on Java 8 as well, and we want to release it to everyone.